app/lib/lnd/methods/walletController.js

```diff
@@ -140,15 +140,15 @@ export function unlockWallet(walletUnlocker, { wallet_password }) {
  */
 export function initWallet(
   walletUnlocker,
-  { wallet_password, cipher_seed_mnemonic, aezeed_passphrase = '' }
+  { wallet_password, cipher_seed_mnemonic, aezeed_passphrase = '', recovery_window }
 ) {
   return new Promise((resolve, reject) => {
     walletUnlocker.initWallet(
       {
         wallet_password: Buffer.from(wallet_password),
         cipher_seed_mnemonic,
         aezeed_passphrase: Buffer.from(aezeed_passphrase, 'hex'),
-        recovery_window: 250
+        recovery_window
```

Comment:
no need for this right? since we are not allowing the user to configure this and are hardcoding the value in the `recoverOldWallet ` func

Reply:
This is needed because we want to allow the caller of `initWallet` to be able to specify what values gets set as `recovery_window`. 

In `createNewWallet` we don't set it. In `recoverOldWallet` we set it to `250`